tests/formats/epoch_processing/README.md

```diff
@@ -32,9 +32,8 @@ The provided pre-state is already transitioned to just before the specific sub-t
 
 Sub-transitions:
 
-Sub-transitions:
-
 - `justification_and_finalization`
+- `inactivity_penalty_updates`
 - `rewards_and_penalties`
 - `registry_updates`
 - `slashings`
```

Comment:
```suggestion
- `slashings`
- `sync_committee_updates`
```

Reply:
ah actually this should be at the end of this column, but github won't let me comment down there